app/javascript/components/public-dashboard/profile.vue

```diff
@@ -30,35 +30,25 @@
 </template>
 
 <script>
-import { mapState } from 'vuex';
+import { mapGetters } from 'vuex';
 
-import { CREATE_PUBLIC_USER_ENTRY } from '../store/mutation-types';
-import { COMPUTE_USERS_ORGANIZATION_WIDE_SCORE } from '../store/action-types';
+import { COMPUTE_USERS_ORGANIZATION_WIDE_SCORE } from '../../store/action-types';
+import { DEFAULT_USER_DATA } from '../../store/modules/public-dashboard/state';
 
 export default {
   props: {
     organizationId: String,
     githubLogin: String,
     imageUrl: String,
   },
-  created() {
-    this.$store.commit(CREATE_PUBLIC_USER_ENTRY, this.githubLogin);
-    this.$store.dispatch(
-      COMPUTE_USERS_ORGANIZATION_WIDE_SCORE, {
-        githubUserLogin: this.githubLogin,
-        organizationId: this.organizationId,
-      }
-    );
-  },
   computed: {
     githubUrl() {
       return `https://github.com/${this.githubLogin}`;
     },
-    ...mapState({
-      userData(state) {
-        return state.publicDashboard.mapGithubUserToScores[this.githubLogin];
-      },
-    }),
+    ...mapGetters(['_userData']),
+    userData() {
+      return this._userData(this.githubLogin);
```

Comment:
Unexpected dangling '_' in '_userData'                             no-underscore-dangle